src/api/spec/controllers/source/key_info_controller_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Source::KeyInfoController, type: :controller do
+  describe 'GET #show' do
+    let(:user) { create(:confirmed_user) }
+    let(:project) { create(:project, name: "test_project", title: "Test Project") }
+    let(:backend_url) { CONFIG['source_url'] + Project::KeyInfo.send(:backend_url, project.name) }
+    let(:gpg_public_key) { Faker::Lorem.characters(1024) }
+    let(:ssl_certificate) { Faker::Lorem.characters(1024) }
+    let(:keyinfo_response) do
+      %(<keyinfo project="Test"><pubkey algo="rsa">#{gpg_public_key}</pubkey><sslcert>#{ssl_certificate}</sslcert></keyinfo>)
+    end
+
+    before do
+      Rails.cache.clear
+      stub_request(:get, backend_url).and_return(body: keyinfo_response)
```

Comment:
hmmm ok that might be a problem for these controller specs, since the obs signer is not setup by default in the vagrant machine so it won't able to return a pubkey or ssl cert...